config/transformers/shadowToCss.ts

```diff
@@ -0,0 +1,44 @@
+import {toHex} from 'color2k'
+import StyleDictionary from 'style-dictionary'
+import {ShadowTokenValue} from '../../types/ShadowTokenValue'
+import {isShadow} from '../filters/isShadow'
+import {alpha} from '../utilities'
+
+/**
+ * checks if all required properties exist on shadow token
+ * @param object - ShadowTokenValue
+ * @returns void or throws error
+ */
+const checkForShadowTokenProperties = (shadow: ShadowTokenValue) => {
+  const requiredShadowProperties = ['color', 'offsetX', 'offsetY', 'blur', 'spread']
+  for (const prop of requiredShadowProperties) {
+    if (prop in shadow === false) throw new Error(`Missing propery: ${prop} on shadow token ${JSON.stringify(shadow)}`)
```

Comment:
nitpicking, but this is a little long for shorthand conditional statement, can we go with more verbose for improved readability? It makes it easier to tell you're short-circuiting the function.

```suggestion
    if (prop in shadow === false) {
      throw new Error(`Missing propery: ${prop} on shadow token ${JSON.stringify(shadow)}`)
    }
```